ldap/servers/slapd/log.c

```diff
@@ -418,6 +429,10 @@ g_log_init()
     if ((loginfo.log_error_rwlock = slapi_new_rwlock()) == NULL) {
         exit(-1);
     }
+    loginfo.log_error_buffer = log_create_buffer(LOG_BUFFER_MAXSIZE);
+    if ((loginfo.log_error_buffer->lock = PR_NewLock()) == NULL) {
```

Comment:
Should also test loginfo.log_error_buffer != NULL

Reply:
> Should also test loginfo.log_error_buffer != NULL

Perhaps, again this was copy and paste code.  So it's been working for a long time, but I'll take a look at it today.

Reply:
Hmm, if log_error_buffer would be NULL we would already crash in log_create_buffer().  Since this happens at server startup I don't think this is a necessary check as the server would be having all kinds of problems prior to hitting this code :-)